src/utils/session.ts

```diff
@@ -14,13 +13,13 @@ const storageSession = (browser.storage as any)?.session;
  * Stores the password key in the session storage.
  * Extension only.
  */
-export async function startSession(encryptionData: IEncryptionData, timeout: number) {
+export async function startSession(encryptionKey: CryptoKey, timeout: number) {
   if (IS_EXTENSION && !IS_OFFSCREEN_TAB) {
     browser.runtime.connect({ name: CONNECTION_TYPES.SESSION });
     const sessionExpires = Date.now() + timeout;
     await storageSession.set(
       {
-        encryptionData: await exportEncryptionData(encryptionData),
+        encryptionKey: await exportEncryptionKey(encryptionKey),
```

Comment:
This line is little bit quirky. Both the property and the constant has the same name: `encryptionKey` which suggests that they are with the same type, but I see you need to us `exportEncryptionKey` function to fix the data type. I suggest to differentiate the naming to make it more clear what is the difference between those 2.